plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala

```diff
@@ -246,14 +246,13 @@ object ScalafmtPlugin extends AutoPlugin {
         )
     }
 
-  private def trueOrBoom(analysis: ScalafmtAnalysis): Boolean = {
+  private def trueOrBoom(analysis: ScalafmtAnalysis): Unit = {
```

Comment:
```suggestion
  private def throwOnFailures(analysis: ScalafmtAnalysis): Unit = {
```

Reply:
added in #168. merging this one for now.